source/Cosmos.HAL2/Cosmos.HAL2.csproj

```diff
@@ -9,15 +9,15 @@
     </PropertyGroup>
 
     <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
-      <AllowUnsafeBlocks>True</AllowUnsafeBlocks>
+      <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
I think we should be fine to just have a single PropertyGroup for all options, since the options shouldnt change. Please remove all but the first PropertyGroup